lib/sinon/match.js

```diff
@@ -8,7 +8,6 @@
  */
 "use strict";
 
-var create = require("./util/core/create");
 var deepEqual = require("./util/core/deep-equal").use(match); // eslint-disable-line no-use-before-define
 var every = require("./util/core/every");
```

Comment:
Yeah, I remember we came to this conclusion when we talked about Array-like objects and Maps and Sets.

In [this comment](https://github.com/sinonjs/sinon/pull/1215#issuecomment-269551320) I talk about it. Actually this method is really useful when it comes to iterating through `Map` and `Set` objects in order to be able to check all of their elements for our related matchers.